### PR TITLE
Pytest checkers

### PR DESCRIPTION
**Pull request by Friz-zy**
Optional checkers:
- execute `py.test -q`
- execute `pytest -q --cov`

### CODE REVIEW

**Comment by alexcouper** [comment]
Thanks for the PRs - I'll do a review of them both now...

Seems to be a test failing on travis [here](https://travis-ci.org/alexcouper/captainhook/jobs/79169948#L271). 

May well be distribution related. I'm running on a mac and they pass - might be you get different output back from git on a linux distribution. Not sure.

File: features/pytest_cov_tox.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Feature: pytest-cov obeys tox
+
+    Background:
+        Given that I am in a git repository
+        And I have installed captainhook
+        And I have a tox.ini file with pytest-cov=on;0
+
+
+    Scenario: When a pytest-cov is present in tox.ini it should be obeyed.
+       When I create a file in "." called "test_one.py" containing a line over 80 chars
```
Comment (alexcouper) [inline features/pytest_cov_tox.feature]: I'm not sure this is the right test for this case. I expected to see this containing a test that created a failing test, or that created some code that wasn't tested and confirmed in either case that we did get an error.

**Comment by Friz-zy** [comment]
Yep, thanks for feedback, i've been waiting for it.
About dependencies: i planned this modules as optional so i don't add pylint and pytest to requirements, but i can do it if you think that it's necessary.
About refactoring and tests: i don't like repeat myself, so i'll refactor it. But it may take a time...

**Comment by alexcouper** [comment]
Yeah i like it being optional - and it won't be installed when people pip install, it's just for us to be able to run any sort of behave-like test we'll need it in test-requirements.

Nice work though :)

**Comment by alexcouper** [comment]
I'll wait for this to get rebased, green and merged before releasing a new captainhook version.

**Comment by Friz-zy** [comment]
Thanks, I'm trying to figure out what is wrong and how reproduce it

**Comment by Friz-zy** [comment]
Merge it please before next conflict with master =)

**Comment by alexcouper** [comment]
@Friz-zy ah hadn't seen that you had resolved this.

I'll take a look at it tomorrow.

**Comment by alexcouper** [comment]
Nice one @Friz-zy :) I'll release a new version shortly...

**Comment by alexcouper** [comment]
Released 0.8.6.

**Pull request merged**